controllers/util/version.go

```diff
@@ -0,0 +1,125 @@
+package util
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	semver "github.com/blang/semver/v4"
+	configv1 "github.com/openshift/api/config/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	// TODO: update the fixed in kernel versions
+	// supported kernel version corresponding to the OCP version
```

Comment:
Why would you need to know/check/use the OCP version?

Reply:
Can we resolve this?